### ISSUE TITLE
Usefulness of Heppy

### ISSUE THREAD

**Issue by kreczko** [BristolTopGroup/NTupleProduction#147]
Probably worth having a *quick* look at Heppy since it is part of CMSSW:
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideHeppy
https://github.com/cbernet/cmssw/tree/heppy_integration_7_0_9_patch2/PhysicsTools/Heppy
Initial presentation:
http://indico.cern.ch/event/346174/contribution/1/material/slides/1.pdf

Code:
https://github.com/HEP-FCC
https://github.com/HEP-FCC/heppy (core)
https://github.com/HEP-FCC/heppy_fcc

Basically I am looking for a way to produce our ntuples in a more pythonic way (with C++ underneath), like it used to be the case for the ConfigurableAnalysis at the beginning of Run1. Hopefully without the drawbacks of that package.

**Comment by kreczko** [comment]
Preliminary report:
 - good ideas
 - outdated documentation
 - setup non-trivial
 - does not work with cmsRun (has its own binary) -> more difficult for CRAB

Recommendation: Wait until HEPPy matures, preferably do not use CMSSW bundled version
